### PR TITLE
21-07-2018_PS changes in UI

### PR DESCRIPTION
**Pull request by prashantsalunke**
@anushashetty Made changes for:
1. alerts counter UI issue
2. Fixed few broken UI issues

### CODE REVIEW

File: application/modules/frontend/views/default/hover-nav.php [hunk 1]
```diff
@@ -329,7 +329,9 @@ class="hover" alt=""
 					<a href="<?php echo $myStationUrl; ?>" style="opacity: 1 !important;position: relative;z-index: 11;">
 						<img src="<?php echo asset_url(); ?>/images/ts/Alerts1.png" style="width: 50px;">
 					</a>
-					<span style="background: red none repeat scroll 0 0;border: 1px solid #fff;border-radius: 100px;box-shadow: 0 0 6px #aaa;left: -30px;position: absolute;text-align: left;top: 20px;width: 50px;z-index: 1;" class="badge"><?php echo $totalcount; ?></span>
+
+					<span style="background: red none repeat scroll 0 0;border: 1px solid #fff;border-radius: 100px;box-shadow: 0 0 6px #aaa;left: -30px;position: absolute;text-align: left;top: 20px;width: 50px;z-index: 1;" class="badge"><?php $totalcount=$this->common->getTotalCountForAlert(['busi_id']);
```
Comment (rajeshbakade65) [inline application/modules/frontend/views/default/hover-nav.php]: @prashantsalunke remove this code and add this to respective controller::action() .

Comment (rajeshbakade65) [reply application/modules/frontend/views/default/hover-nav.php]: Also add proper busi_id from session as a parameter in the function call.

File: application/modules/frontend/views/default/inner-header.php [hunk 2]
```diff
@@ -294,14 +294,14 @@
...
+                                 <?php if(is_null($this->session->userdata('tsuserid'))) {  ?>
 									<a onclick="javascript:login_message()" style="opacity: 1 !important;position: relative;z-index: 11;">
 								<?php } else { ?>
 									<a href="<?php echo $myStationUrl; ?>" style="opacity: 1 !important;position: relative;z-index: 11;">
 								<?php } ?>
 								<img src="<?php echo base_url();?>assets/images/ts/Alerts1.png" style="width: 50px;">
 								<h5>My Alerts</h5></a>
-                                 <span style="background: red none repeat scroll 0 0;border: 1px solid #fff;border-radius: 100px;box-shadow: 0 0 6px #aaa;left: -3px;position: absolute;text-align: left;top: 20px;width: 50px;z-index: 1;" class="badge"><?php echo $totalcount; ?></span>
+                                 <span style="background: red none repeat scroll 0 0;border: 1px solid #fff;border-radius: 100px;box-shadow: 0 0 6px #aaa;left: -3px;position: absolute;text-align: left;top: 20px;width: 50px;z-index: 1;" class="badge"><?php  $totalcount=$this->common->getTotalCountForAlert(['busi_id']); echo $totalcount; ?></span>
```
Comment (rajeshbakade65) [inline application/modules/frontend/views/default/inner-header.php]: @prashantsalunke remove this code and add this to respective controller::action() .
Also add proper busi_id from session as a parameter in the function call.

Comment (rajeshbakade65) [reply application/modules/frontend/views/default/inner-header.php]: @shettyanusha16 please check and add your comment 
Thanks

File: application/modules/frontend/views/default/hover-nav.php (hunk 1 above)
Comment (rajeshbakade65) [reply application/modules/frontend/views/default/hover-nav.php]: @shettyanusha16 please check and add your comment 
Thanks

File: application/modules/frontend/views/default/inner-header.php (hunk 2 above)
Comment (shettyanusha16) [reply application/modules/frontend/views/default/inner-header.php]: @prashantsalunke : Rajesh has pointed out correctly. Please make the changes.

File: application/modules/frontend/views/default/hover-nav.php (hunk 1 above)
Comment (shettyanusha16) [reply application/modules/frontend/views/default/hover-nav.php]: @prashantsalunke : Rajesh has pointed out correctly. Please make the changes.

**Pull request merged**